UI/admin_page.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Maintenance Tracker | Repairs | Online dignostics</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link 
+      rel="stylesheet" type="text/css" media="screen" href="styles/main.css" 
+    />
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+</head>
+</head>
+<body>
+  <header>
+    <h1>MaintenanceTracker</h1>
+    <p>
+      The Web Application for all your diagnostics, repairs, 
+      online consultations. Soutions for hardware, software &amp; 
+      firmaware issues.
+    </p>
+    <nav>
+      <a href="index.html">Sign Out</a> |
+      <a href="signin.html">Requests</a>
+      <a href="#">
+        <img src="images/user_avatar_sm-46px.png" alt="profile picture">
+      </a>
+    </nav>
+  </header>
+  <hr>
+  <article>
+    <div>
+      <h2>Admins Log</h2>
+        <h4>Active Requests</h4>
+      <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras pretium, dui sit amet rutrum consectetur, augue magna fermentum odio, dictum posuere diam ante eget tellus. Nunc quam est, feugiat nec ex nec, malesuada semper eros. Morbi tincidunt aliquam pretium. Sed ac cursus arcu. 
+      </p>
+      
+    </div>   
+  </article>
+  <div class="long-x-axis">
```

Comment:
Add a Search box and filter (drop list?) - Admin should be able to search and filter requests